### PR TITLE
WQP-1176 and WQP-1178 ETL tweaks

### PR DESCRIPTION
**Pull request by skaymen**

### CODE REVIEW

File: transformStation.sql [hunk 1]
```diff
@@ -14,8 +14,7 @@ delete from wqx_station_local
                           left join wqx.organization org
                             on monitoring_location.org_uid = org.org_uid
                     where wqx_station_local.station_id = monitoring_location.mloc_uid and
-                          org.org_id not like '%TEST%' and
```
Comment (jkreft-usgs) [inline transformStation.sql]: You'll need to use org.org_uid (numeric primary key) instead of org.org_id (text)

Comment (skaymen) [reply transformStation.sql]: Fixed.

File: transformActivity.sql [hunk 2]
```diff
@@ -56,10 +56,10 @@ select /*+ parallel(4) */
        activity.act_uid activity_id,
        activity_type.actyp_cd activity_type_code,
        activity_media_subdivision.amsub_name activity_media_subdiv_name,
-       nvl(to_char(activity.act_start_time, 'hh24:mi:ss'), '00:00:00') activity_start_time,
+       nvl(to_char(activity.act_start_time, 'hh24:mi:ss'), null) activity_start_time,
```
[the remark this answers is not in the document]
Comment (skaymen) [reply transformActivity.sql]: Done.

File: transformActivity.sql [hunk 3]
```diff
@@ -56,10 +56,10 @@ select /*+ parallel(4) */
...
        start_time_zone.tmzone_cd act_start_time_zone,
        to_char(activity.act_end_date, 'yyyy-mm-dd') activity_stop_date,
-       nvl2(act_end_date, nvl(to_char(activity.act_end_time, 'hh24:mi:ss'), '00:00:00'), null) activity_stop_time,
+       nvl2(act_end_date, nvl(to_char(activity.act_end_time, 'hh24:mi:ss'), null), null) activity_stop_time,
```
[the remark this answers is not in the document]
Comment (skaymen) [reply transformActivity.sql]: Done.

File: transformActivity.sql [hunk 4]
```diff
@@ -103,23 +103,23 @@ select /*+ parallel(4) */
...
          when activity.act_sam_collect_meth_id is not null and
               activity.act_sam_collect_meth_context is not null
            then activity.act_sam_collect_meth_name
-         else 'USEPA'
+         else null
        end sample_collect_method_name,
        activity.act_sam_collect_meth_qual_type,
        activity.act_sam_collect_meth_desc,
-       nvl(sample_collection_equip.sceqp_name, 'Unknown') sample_collect_equip_name,
+       nvl(sample_collection_equip.sceqp_name, null) sample_collect_equip_name,
```
[the remark this answers is not in the document]
Comment (skaymen) [reply transformActivity.sql]: Done.

File: transformStation.sql [hunk 5]
```diff
@@ -87,8 +85,7 @@ delete from wqx_station_local
        not exists (select null
                      from station_no_source
                     where wqx_station_local.station_id = station_no_source.station_id and
-                          organization not like '%TEST%' and
-                          organization not like '%TRAINING%');
+                          organization not between 2000 and 2999;
```
[the remark this answers is not in the document]
Comment (skaymen) [reply transformStation.sql]: Fixed.

File: transformStation.sql [hunk 6]
```diff
@@ -106,8 +103,7 @@ merge into wqx_station_local o
                     geom
                from station_no_source
               where station_no_source.site_id not in (select site_id from wqx_station_local where station_source = 'WQX') and
-                    organization not like '%TEST%' and
-                    organization not like '%TRAINING%'
+                    organization not between 2000 and 2999;
```
[the remark this answers is not in the document]
Comment (skaymen) [reply transformStation.sql]: Fixed.

File: transformStation.sql [hunk 7]
```diff
@@ -205,10 +201,10 @@ select 3 data_source_id,
                monitoring_location.mloc_latitude latitude,
                monitoring_location.mloc_longitude longitude,
                cast(monitoring_location.mloc_source_map_scale as varchar2(4000 char)) map_scale,
-               nvl(horizontal_collection_method.hcmth_name, 'Unknown') geopositioning_method,
-               nvl(horizontal_reference_datum.hrdat_name, 'Unknown') hdatum_id_code,
+               nvl(horizontal_collection_method.hcmth_name, null) geopositioning_method,
```
[the remark this answers is not in the document]
Comment (skaymen) [reply transformStation.sql]: Fixed.

File: transformStation.sql [hunk 8]
```diff
@@ -205,10 +201,10 @@ select 3 data_source_id,
...
+               nvl(horizontal_reference_datum.hrdat_name, null) hdatum_id_code,
                monitoring_location.mloc_vertical_measure elevation_value,
-               nvl2(monitoring_location.mloc_vertical_measure, nvl(measurement_unit.msunt_cd, 'ft'), null) elevation_unit,
+               nvl2(monitoring_location.mloc_vertical_measure, nvl(measurement_unit.msunt_cd, null), null) elevation_unit,
```
[the remark this answers is not in the document]
Comment (skaymen) [reply transformStation.sql]: Fixed.

File: transformStation.sql [hunk 9]
```diff
@@ -205,10 +199,10 @@ select 3 data_source_id,
                monitoring_location.mloc_latitude latitude,
                monitoring_location.mloc_longitude longitude,
                cast(monitoring_location.mloc_source_map_scale as varchar2(4000 char)) map_scale,
-               nvl(horizontal_collection_method.hcmth_name, 'Unknown') geopositioning_method,
-               nvl(horizontal_reference_datum.hrdat_name, 'Unknown') hdatum_id_code,
+               horizontal_collection_method.hcmth_name geopositioning_method,
+               horizontal_reference_datum.hrdat_name, null) hdatum_id_code,
```
Comment (dsteinich) [inline transformStation.sql]: ", null)" needs to be deleted too.

**Pull request merged**